x/bep3/types/params.go

```diff
@@ -40,8 +44,10 @@ var (
 type Params struct {
 	BnbDeputyAddress  sdk.AccAddress `json:"bnb_deputy_address" yaml:"bnb_deputy_address"`     // Bnbchain deputy address
 	BnbDeputyFixedFee uint64         `json:"bnb_deputy_fixed_fee" yaml:"bnb_deputy_fixed_fee"` // Deputy fixed fee in BNB
-	MinBlockLock      uint64         `json:"min_block_lock" yaml:"min_block_lock"`             // AtomicSwap minimum block lock
-	MaxBlockLock      uint64         `json:"max_block_lock" yaml:"max_block_lock"`             // AtomicSwap maximum block lock
+	MinAmount         uint64         `json:"min_amount" yaml:"min_amount"`                     // Minimum swap amount
```

Comment:
I would use sdk.Int for min/max amount unless there is a good reason not to. Since incoming swap amounts are already sdk.Int, it makes comparison easier and avoids overflow edge cases.

Reply:
https://github.com/Kava-Labs/kava/pull/546/commits/5e3c64eaf8f9ea854ae1bedbf594e2a3309af620

Reply:
Also updated BnbDeputyFixedFee